tslint.json

```diff
@@ -13,7 +13,8 @@
             "call-signature",
             "property-declaration"
         ],
-        "max-classes-per-file":false,
+        "align": false,
```

Comment:
it throws errors on master code and it looked strange after a fix, so currently, just disabled it